### PR TITLE
Adding better formatting to publish dialog for summaries.

### PR DESCRIPTION
**Pull request by Liametc**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,10 @@ python:
 env:
     - SHOTGUN_EXTERNAL_REPOS_ROOT=/tmp
 
+dist: xenial
```
Comment (j0yu) [inline .travis.yml]: Submit this back to Shotgun, adding the URL for official Travis Docs

Remove the commented out bits later on in this file

File: python/tk_multi_publish2/publish_tree_widget/tree_node_item.py [hunk 2]
```diff
@@ -60,7 +60,7 @@ def __repr__(self):
     def __str__(self):
         return "%s %s" % (self._item.type_display, self._item.name)
 
-    def create_summary(self):
+    def create_summary(self, level=0):
         """
         Creates summary of actions
```
Comment (j0yu) [inline python/tk_multi_publish2/publish_tree_widget/tree_node_item.py:65]: Add docs for `level`

**Review by j0yu — CHANGES_REQUESTED** [review]
Maybe add a screenshot to this PR of the HTML changes

Personally, I think all this can probably be suggested back to Shotgun and let them choose whether they like it or not

**Comment by Liametc** [comment]
image

**Comment by Liametc** [comment]
Just removed the travis fix, to make it it's own PR allowing me to separate this out and push to shotgun. this will fail the tests in the meantime until that pr is merged

**Pull request closed without merging**